yodeploy/virtualenv.py

```diff
@@ -22,19 +16,26 @@ def sha224sum(filename):
     return m.hexdigest()
 
 
-def get_python_version(compat):
-    """Return the version of Python that an app will use.
-
-    Based on the compat level.
-    """
-    if compat < 5:
+def get_python_version(compat=None, is_deploy=False):
+    python3 = shutil.which('python3')
+    if not python3:
+        log.error(
+            'python3 not found in PATH%s' % (
+                'for deploy VE' if is_deploy else ''
```

Comment:
'for deploy VE' -> ' for deploy VE'

Reply:
do you mean to add a space before `for` here?

Reply:
yup